libp2p/protocols/pubsub/pubsubpeer.nim

```diff
@@ -83,12 +79,13 @@ proc sendObservers(p: PubSubPeer, msg: var RPCMsg) =
 proc handle*(p: PubSubPeer, conn: Connection) {.async.} =
   logScope:
     peer = p.id
+
   debug "starting pubsub read loop for peer", closed = conn.closed
   try:
     try:
       while not conn.atEof:
         trace "waiting for data", closed = conn.closed
-        let data = await conn.readLp(64 * 1024).wait(DefaultReadTimeout)
+        let data = await conn.readLp(64 * 1024)
```

Comment:
is it fine to remove the timeout here?

Reply:
Yes, this was intentional, I don't think we need one there.